Sources/VitalCore/Core/Client/Data Models/Base Models/VitalResource.swift

```diff
@@ -1,4 +1,37 @@
 public enum VitalResource: Equatable, Hashable, Codable {
+  public func resourceToBackfillType() -> BackfillType {
```

Comment:
```suggestion
  @_spi(VitalSDKInternals)
  public func resourceToBackfillType() -> BackfillType {
```

Reply:
The @_spi prevent me from accessing this in `historicalStartDate` function

Reply:
Similarly the @_spi and BackfillType struct causes issue in the model definitions

Reply:
You will need to update the import in the function’s file with `@_spi(VitalSDKInternals) import VitalCore`